src/client/opamAdminCommand.ml

```diff
@@ -761,10 +761,17 @@ let check_command cli =
         are supposed to be available. By default, all checks are run."
   ]
   in
-  let ignore_test_arg =
-    OpamArg.mk_flag ~cli OpamArg.cli_original ["ignore-test-doc";"i"]
-      "By default, $(b,{with-test}) and $(b,{with-doc}) dependencies are \
-       included. This ignores them, and makes the test more tolerant."
+  let with_test_arg =
+    OpamArg.mk_flag ~cli OpamArg.(cli_from cli2_4) ["with-test"]
```

Comment:
```suggestion
    OpamArg.mk_flag ~cli OpamArg.(cli_from cli2_4) ["t";"with-test"]

Reply:
Adding short forms to be select to have them all more easily `-tds` instead of `--with-test --with-doc --with-dev-setup`